CI/travis/make_linux

```diff
@@ -8,6 +8,16 @@ handle_default() {
 	mkdir -p build
 	cd build
 
+	# Do a install, to test things, and make sure that Cmake's
+	# package manager can find things on system path later
+	# then clean, so we can re-do things with the appropriate cmake
+	# flags, and sudo creates files as root, that make package can not
+	# write over...
+	cmake ../
+	make
+	sudo make install
+	rm * -rf
```

Comment:
coming back to this;
i don't know if this is the best fix here;
it may be a good idea to try to pass an LD_LIBRARY_PATH or some other lib-path inside `bindings/python/setup.py.cmakein`
again, i'd ask @tfcollins to take a look and maybe consider a solution for this;

if we fix it here [in our build], others trying to build this manually might run into this and open issues;

if [however] this is the only possible fix, i would do something like:
```
if [ "$TRAVIS" = "true" ] ; then
	cmake ../
	make
	sudo make install
	rm * -rf
fi
``` 
this is also a bit of a fail-safe switch;

thing is: i run this script locally before doing patches [to these build-scripts]; saves some time during development; so the `sudo make install` installs things on the host, which i would avoid doing;
i also use some dockers, but sometimes it's quicker without them, since [currently] the build doesn't modify much on the host [other than the APT packages that it installs]

Reply:
I did talk about this on the phone with @tfcollins ; and this is what we came to the conclusion was easier/best. The python parts need to get deployed automically to pypi on a release, and (in theory) they should now.

It does seem to build fine, it's just alot of annoying output. It also tests that "make install" works properly, and should let us build the examples in CI as well. (as a build test for more things).

Reply:
I really have no idea why CPack doesn't error out. However, I would not package the bindings into the deb/rpm/tar.

Can cpack ignore files during packaging?

Reply:
I think the issue is the way that the python bindings get added to the CODE

and we should be doing something like this in the install (splitting things)
```
install(TARGET app DESTINATION ... COMPONENT applications)
install(TARGET library DESTINATION ... COMPONENT libraries)
install(FILES <headers> DESTINATION ... COMPONENT headers)
```

and then in the `cmake/LinuxPackaging.cmake`
```
set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
set(CPACK_COMPONENTS_ALL applications)
include(CPack)
```

see [StackOverflow](https://stackoverflow.com/questions/27619234/how-to-exclude-files-dirs-from-cpack-binary-using-cmake) and [wiki](https://gitlab.kitware.com/cmake/community/-/wikis/doc/cpack/Component-Install-With-CPack)